index.js

```diff
@@ -0,0 +1,266 @@
+const fs = require("fs");
+const axios = require("axios");
+const Image = require("ascii-art-image");
+const inquirer = require("inquirer");
+const color = require("colors-cli/safe");
+const errorColor = color.red.bold;
+const addedColor = color.green.bold;
+const menuColor = color.blue.bold;
+const listColor = color.magenta.bold;
+const deletedColor = color.yellow.bold;
+const personalMessageColor = color.green_b.bold;
+
+const args = process.argv;
+
+// The "index.js" is 8 characters long so -8 removes last 8 characters
+const currentWorkingDirectory = args[1].slice(0, -8);
+
+if (!fs.existsSync(currentWorkingDirectory + "todo.txt")) {
+  let createStream = fs.createWriteStream("todo.txt");
+  createStream.end();
+}
+
+const showInteractiveMenu = () => {
+  inquirer
+    .prompt([
+      {
+        type: "input",
+        name: "name",
+        message: "What's your name?",
+      },
+      {
+        type: "list",
+        name: "menu",
+        message: "What you would like to do next?",
+        choices: [
+          "Create new note",
+          "Show all notes",
+          "Delete note",
+          "Show menu",
+        ],
+      },
+    ])
+    .then((answers) => {
+      if (answers.menu === "Create new note") {
+        inquirer
+          .prompt([
+            {
+              type: "input",
+              name: "new_note",
+              message: `So, ${answers.name}, what's on your mind?`,
+            },
+          ])
+          .then((answers) => {
+            addNewToDo(answers.new_note);
+          })
+          .catch((error) => {
+            console.log(error);
+          });
+      } else if (answers.menu === "Show all notes") {
+        console.log(
+          personalMessageColor(
+            `Hey, ${answers.name}! Those are all your notes for now:`
+          )
+        );
+        getAllToDo();
+      } else if (answers.menu === "Delete note") {
+        inquirer
+          .prompt([
+            {
+              type: "input",
+              name: "delete",
+              message: `So, ${answers.name}, which note to delete?`,
+            },
+          ])
+          .then((answers) => {
+            deleteToDo(answers.delete);
+          })
+          .catch((error) => {
+            console.log(error);
+          });
+      } else if (answers.menu === "Show menu") {
+        console.log(
+          personalMessageColor(
+            `Hey, ${answers.name}! Please choose one of the following:`
+          )
+        );
+        getHelpInfo();
+      } else {
+      }
+    })
+    .catch((error) => {
+      console.log(error);
+    });
+};
+
+const isInputNumber = (inputNumber) => {
+  return Number.isInteger(Number(inputNumber)) ? true : false;
+};
+
+const fetchFromApi = async (pokemonId) => {
+  const pokemonName = await getPokemon(pokemonId);
+
+  return typeof pokemonName !== "undefined" ? pokemonName : undefined;
+};
+
+const getPokemon = async (pokemonId) => {
+  try {
+    const response = await axios(
```

Comment:
why axios? you could simply use  await fetch.prefer native/built-in functionality rather than using external packages.